distributed/utils_test.py

```diff
@@ -531,6 +533,22 @@ def client(loop, cluster_fixture):
         yield client
 
 
+@pytest.fixture
+def client_no_amm(client):
+    """Let a sync test that relies on the Active Memory Manager (AMM) to run wether
```

Comment:
This docstring is confusing me. Isn't this actively turning off AMM? Also, do we need the `assert not client.amm.running()` after `yield client`? Shouldn't we just ensure that AMM is reset to `before`?

Reply:
Revisited the fixture